packages/langium-cli/src/generator/module-generator.ts

```diff
@@ -13,29 +13,48 @@ export function generateModule(grammar: langium.Grammar, config: LangiumConfig):
     const node = new CompositeGeneratorNode();
     node.append(generatedHeader);
     if (config.langiumInternal) {
+        node.append(`import { LanguageMetaData, ${config.parserConfig ? 'ParserConfig' : 'defaultParserConfig'} } from '../..';`, NL);
         node.append("import { Module } from '../../dependency-injection';", NL);
         node.contents.push("import { LangiumGeneratedServices, LangiumServices } from '../../services';", NL);
     } else {
-        node.append("import { LangiumGeneratedServices, LangiumServices, Module } from 'langium';", NL);
+        node.append(`import { LangiumGeneratedServices, LangiumServices, LanguageMetaData, Module, ${config.parserConfig ? 'ParserConfig' : 'defaultParserConfig'} } from 'langium';`, NL);
     }
     node.append(
         'import { ', grammar.name, "AstReflection } from './ast';", NL,
         "import { grammar } from './grammar';", NL, NL
     );
 
-    node.append('export const languageMetaData = {', NL);
+    node.append('export const languageMetaData: LanguageMetaData = {', NL);
     node.indent(metaData => {
         metaData.append(`languageId: '${config.languageId}',`, NL);
         metaData.append(`fileExtensions: [${config.fileExtensions && config.fileExtensions.map(e => appendQuotesAndDot(e)).join(', ')}]`, NL);
     });
     node.append('};', NL, NL);
 
+    if (config.parserConfig) {
+        node.append('export const parserConfig: ParserConfig = {', NL);
+        node.indent(configNode => {
+            const chevrotainConfig = config.parserConfig!.chevrotainConfig;
+            configNode.append('chevrotainConfig: {', NL);
+            configNode.indent(chevrotainConfigNode =>
+                Object.keys(chevrotainConfig).forEach(key => {
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    const value = (chevrotainConfig as any)[key];
+                    chevrotainConfigNode.append(`${key}: ${typeof value === 'string' ? `'${value}'` : value},`, NL);
+                })
+            );
+            configNode.append('}', NL);
+        });
+        node.append('};', NL, NL);
+    }
+
     node.append('export const ', grammar.name, 'GeneratedModule: Module<LangiumServices, LangiumGeneratedServices> = {', NL);
     node.indent(moduleNode => {
         moduleNode.append(
             'Grammar: () => grammar(),', NL,
             'AstReflection: () => new ', grammar.name, 'AstReflection(),', NL,
-            'LanguageMetaData: () => languageMetaData', NL
+            'LanguageMetaData: () => languageMetaData,', NL,
+            `ParserConfig: () => ${config.parserConfig ? 'parserConfig' : 'defaultParserConfig'}`, NL
```

Comment:
I'd propose a different approach: Mark the `ParserConfig` service as optional and merge the default options with the user defined options in the ChevrotainWrapper constructor:
```typescript
    constructor(tokens: TokenType[], config?: IParserConfig = {}) {
        const mergedConfig = {
            recoveryEnabled: true,
            nodeLocationTracking: 'onlyOffset',
            ...config
        };
        super(tokens, mergedConfig);
    }
```